src/components/PlayerCardContainer/CustomInput.js

```diff
@@ -0,0 +1,11 @@
+import React, {useState} from "react";
+
+import './CustomInput.css'
+
+function CustomInput(props) {
+        return (
+            <input type={props.type} className={props.classes} placeholder={props.placeHolder} onChange={props.onNameChange ?  props.onNameChange : props.onBalanceChange} />
```

Comment:
Make this more generic, just take in a handler for onChange, the name doesn't matter (changeHandler or something instead of onNameChange and onBalanceChange)